diff --git src/pydap/net.py src/pydap/net.py
index b8d82e7..1952067 100644
--- src/pydap/net.py
+++ src/pydap/net.py
@@ -3,6 +3,7 @@ from webob.exc import HTTPError
 from contextlib import closing
 import requests
 import ssl
+import urllib
 from requests.exceptions import (MissingSchema, InvalidSchema,
                                  Timeout)

@@ -25,6 +26,10 @@ def GET(url, application=None, session=None, timeout=DEFAULT_TIMEOUT,
         _, _, path, query, fragment = urlsplit(url)
         url = urlunsplit(('', '', path, query, fragment))

+    scheme, host, path, query, fragment = urlsplit(url)
+    query = urllib.quote(query)
+    url = urlunsplit((scheme, host, path, query, fragment))
+
     response = follow_redirect(url, application=application, session=session,
                                timeout=timeout, verify=verify)
     # Decode request response (i.e. gzip)
